q2_feature_table/plugin_setup.py

```diff
@@ -101,6 +100,44 @@
                 "result."
 )
 
+_where_description = ("The where parameter takes a SQLite WHERE clause. "
+                      "You can get help forming these expressions at the "
+                      "following links:\n"
+                      "https://en.wikipedia.org/wiki/Where_(SQL)\n"
```

Comment:
Link to table-filtering tutorial: `https://docs.qiime2.org/{{ qiime2.__version__ }}/tutorials/table-filtering.html`